Modelica/Electrical/Analog/Lines/Functions/TestLineZmatrix.mo

```diff
@@ -0,0 +1,64 @@
+within Modelica.Electrical.Analog.Lines.Functions;
+model TestLineZmatrix
```

Comment:
I agree in  general. In fact, CompareCmatrix and PowerLineWIthFence are put in the proper folder Electrical.Analog.Examples.Lines.

However, in case of these particular models (TestLineXmatrix and TestLineCmatrix), which refer just to the newly added functions. I think are better located here, since they just show how these functions work.